util/string_util.cc

```diff
@@ -303,6 +312,31 @@ uint64_t ParseUint64(const std::string& value) {
   return num;
 }
 
+int64_t ParseInt64(const std::string& value) {
+  size_t endchar;
+#ifndef CYGWIN
+  int64_t num = std::stoull(value.c_str(), &endchar);
+#else
+  char* endptr;
+  int64_t num = std::strtoul(value.c_str(), &endptr, 0);
```

Comment:
I believe you should be using `std::strtol` / `std::strtoll` here and line 318.  
Looks good otherwise.

Reply:
good catch!